code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -34,14 +34,11 @@
 	QDEL_NULL(wires)
 	return ..()
 
-/obj/machinery/particle_accelerator/control_box/attack_hand(mob/user)
-	. = ..()
-	if(.)
-		return
-	if(construction_state == PA_CONSTRUCTION_COMPLETE)
-		interact(user)
-	else if(construction_state == PA_CONSTRUCTION_PANEL_OPEN)
+/obj/machinery/particle_accelerator/control_box/multitool_act(mob/living/user, obj/item/I)
+	if(construction_state == PA_CONSTRUCTION_PANEL_OPEN)
 		wires.interact(user)
+		return TRUE
+	return ..()

```

Comment:
`..()` should be called unconditionally at the start of the proc rather than at the end